### PR TITLE
update guzzle lib to same version shopware uses

### PR DESCRIPTION
**Pull request by lhwparis**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "keywords": ["shopware", "s3", "aws"],
     "replace": {
         "league/flysystem": "~1.0",
-        "guzzlehttp/guzzle": "~5.1"
+        "guzzlehttp/guzzle": "~5.3"
```
Comment (mitelg) [inline composer.json:9]: please also update the composer.lock file

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
Hey @lhwparis, thanks for your PR :+1: 
please have a look at my comment :wink:

File: composer.json (hunk 1 above)
Comment (lhwparis) [reply composer.json:9]: done :)

**Review by mitelg — APPROVED** [review]

**Comment by mitelg** [comment]
looks good to me now, waiting for @janbuecker review

**Comment by lhwparis** [comment]
thanks @mitelg :)

**Comment by lhwparis** [comment]
@mitelg @janbuecker any news on merging that?

**Comment by janbuecker** [comment]
Looks good to me. I'll import it and update guzzle to the current implemented Shopware version.

**Pull request closed without merging**